spec/acceptance/config_values_spec.rb

```diff
@@ -0,0 +1,170 @@
+require 'spec_helper_acceptance'
+
+confine(:to, :platform => 'windows')
```

Comment:
You don't need both this confine and "windows_agents" as the latter is functionally equivalent.